sdk/eventgrid/eventgrid/test/internal/util.spec.ts

```diff
@@ -3,7 +3,7 @@
 
 import { assert } from "chai";
 
-import { dateToServiceTimeString } from "../src/util";
+import { dateToServiceTimeString } from "../../src/util";
```

Comment:
Is this utility file only used by internal tests and not the public tests? @ellismg

Reply:
`dateToServiceTimeString` is an internal utility function inside the implementation library that's used by the implementation, but is not part of the public service area.  It's a place where had some core logic that I factored out into a function so I could directly unit test it.  It felt like the tests should be "internal" since this is testing an internal implementation detail of the SDK which may change across versions.

Did I do the right thing here, @KarishmaGhiya?

Reply:
Going to go ahead and merge this now but if I need to fix something up here, let me know.